app/src/main/java/com/moyerun/moyeorun_android/network/calladapter/ApiResult.kt

```diff
@@ -0,0 +1,10 @@
+package com.moyerun.moyeorun_android.network.calladapter
+
+/**
+ * Success : API 호출 성공 시, body를 Wrapping 합니다.
+ * Failure : API 호출 실패 시, errorBody를 Wrpping 합니다.
+ */
+sealed class ApiResult<out T> {
+    data class Success<T>(val body: T) : ApiResult<T>()
+    data class Failure(val exception: Throwable) : ApiResult<Nothing>()
+}
```

Comment:
Failure 주석에 errorBody -> Throwable 로 수정이 필요해보입니다 ㅎ
그리고 중요한 건 아니지만 exception 과 throwable 은 상속관계라도 엄연히 throwable 이 더 큰 범위니까요, exception 보다는 throwable 네이밍이 좋아보입니다.
```suggestion
 * Success : API 호출 성공 시, body를 Wrapping 합니다.
 * Failure : API 호출 실패 시, throwable를 Wrapping 합니다.
 */
sealed class ApiResult<out T> {
    data class Success<T>(val body: T) : ApiResult<T>()
    data class Failure(val throwable: Throwable) : ApiResult<Nothing>()
}
```

Reply:
좋네요오 수정했습니다~!